patches/server/0344-Anti-Xray.patch

```diff
@@ -494,6 +518,7 @@ index 0000000000000000000000000000000000000000..4f3670b2bdb8b1b252e9f074a6af56a0
 +
 +                    // Abuse the obfuscateLayer method to read the blocks of the first layer of the current chunk section
 +                    bitStorageWriter.setBits(0);
++                    random.nextLayer();
```

Comment:
This one can be removed, it doesn't matter tho. I think it feels more correct when it's removed.